.eslintrc.yml

```diff
@@ -6,6 +6,7 @@ plugins:
 parserOptions:
   sourceType: module
 rules:
+  indent: [2, 2]
```

Comment:
Can you make this like the rule below it using the yaml array syntax with error and 2.

Reply:
You got it!